master/usage/configuration/node.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Configuring a node IP address and subnet
+---
+
+Each Calico node has key routing configuration that may be specified in a number of
+different ways.  In particular, this includes the IPv4 and IPv6 addresses and 
+subnets used for routing.
+
+To configure the IP address and subnet of your node, you have the options of:
+-  Set appropriate flags on the `calicoctl node run` command
+-  Set appropriate environments for the `calico/node` container
+-  Configure the node resource using `calicoctl`
+
+> Note that for nodes that were configured prior to v2.1, the subnet information 
+> configured on those nodes will be set to /32 for IPv4 or /128 for IPv6.  If you are
+> planning on using the `cross-subnet IP-in-IP` feature then you must update your node
+> configuration to have the correct subnet information for each node.
+
+#### a) Configure the IP and subnet through runtime args/environments
+
+The IPv4 address and subnet may be explicitly specified using the `--ip` option on 
+`calicoctl node run` or the `IP` environment if you are starting the container
+directly.  For IPv6, use the equivalent `--ip6` option and `IP6` environment.
+
+If you omit the subnet, it is assumed to be /32 for IPv4 and /128 for IPv6 - it is
+recommended to include the subnet information if you specify the IP addresses using
+this approach.
+
+For example (if using calicoctl node run):
+```
+calicoctl node run --ip=10.0.2.10/24
+```
+
+#### b) Autodetect the IP and subnet
+
+The node can be instructed to autodetect the IPv4 address and subnet everytime it
```

Comment:
We should be clear what we mean by "the node" throughout this doc.

I think this line should actually be "The `calico/node` container can be configured to..."